src/Dofs/DofRenumbering.jl

```diff
@@ -217,6 +232,57 @@ function compute_renumber_permutation(dh::DofHandler, _, order::DofOrder.Compone
     return perm
 end
 
+function compute_renumber_permutation(dh::MixedDofHandler, _, order::DofOrder.ComponentWise)
+    field_names = getfieldnames(dh)
+    field_dims = map(fieldname->getfielddim(dh, fieldname), field_names)
+    target_blocks = if isempty(order.target_blocks)
+        collect(Int, 1:sum(field_dims))
+    else
+        if length(order.target_blocks) != sum(field_dims)
+            error("length of target block vector does not match number of components in DofHandler")
+        end
+        order.target_blocks
+    end
+    @assert length(target_blocks) == sum(field_dims)
+    @assert sort!(unique(target_blocks)) == 1:maximum(target_blocks)
+    # Collect all dofs into the corresponding block according to target_blocks
+    nblocks = maximum(target_blocks)
+    dofs_for_blocks = [Set{Int}() for _ in 1:nblocks]
+    component_offsets = pushfirst!(cumsum(field_dims), 0)
+    for fh in dh.fieldhandlers
```

Comment:
Maybe this, with some adjustmens, would work:
```suggestion
    for fh in (dh isa DofHandler ? (dh, ) : dh.fieldhandlers)
```

Reply:
There could sure be some refactoring for better code reuse. I thought it's not really worth it though with the effort to merge the dofhandlers going well.

Reply:
Just thought it would give a smaller diff both now and then, but not worth it if it isn't trivial like in the other method

Reply:
I don't think it's worth it here.

Reply:
Couldn't help myself :p